### ISSUE TITLE
Extension fails to execute build task

### ISSUE THREAD

**Issue by Xevion** [franneck94/Vscode-C-Cpp-Runner#35]
I installed this extension and got it setup. I've read through the documentation and searched for solutions online with no luck.

When I attempt to build the program, I get this back:

```bash
> Executing task:  && C:/ProgramData/chocolatey/bin/gcc.exe -Wall -Wextra -Wpedantic --std=c99 -g3 -O0 -c f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\main.c -o f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\main.o && C:/ProgramData/chocolatey/bin/gcc.exe -Wall -Wextra -Wpedantic --std=c99 -g3 -O0 -c f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\champion.c -o f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\champion.o && C:/ProgramData/chocolatey/bin/gcc.exe -Wall -Wextra -Wpedantic --std=c99 -g3 -O0 f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\main.o f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\champion.o -o f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\outDebug.exe <

&& was unexpected at this time.
The terminal process "C:\Windows\System32\cmd.exe /d /c  && C:/ProgramData/chocolatey/bin/gcc.exe -Wall -Wextra -Wpedantic --std=c99 -g3 -O0 -c f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\main.c -o f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\main.o && C:/ProgramData/chocolatey/bin/gcc.exe -Wall -Wextra -Wpedantic --std=c99 -g3 -O0 -c f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\champion.c -o f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\champion.o && C:/ProgramData/chocolatey/bin/gcc.exe -Wall -Wextra -Wpedantic --std=c99 -g3 -O0 f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\main.o f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\champion.o -o f:\UTSA\Classes\Fall 2021\CS-1714-0D1\Projects\champions\build\Debug\outDebug.exe" terminated with exit code: 1.
```

I looked into it and got the idea it may have been due to the terminal being used by VS Code, so I changed the integrated terminal shell selected from `cmd` to `powershell.exe`. Unfortunately, nothing changed. Search results past this yield no results and I'm now lost about what I should do from here.

```
Windows 10.0.19042
VS Code 1.62.2
C/C++ v1.171
C/C++ Runner v1.5.5
```

**Comment by franneck94** [comment]
I will look into this.
Is your compiler based on a MinGW Compiler?
I am asking because the logic of executing tasks is different for Cygwin and MinGW based compilers on windows.

As a side note in my opinion cmd should work with less issues than the powershell.

**Comment by franneck94** [comment]
I could not reproduce the issue.
I had a similar issue in the past with the VSCodes Python extension.
After a restart of VSCode, it worked again.
Seems to be related to the Task Interface of VSCode's API.

**Comment by franneck94** [comment]
Will be fixed in release 2.0.
